gatsby-ssr.tsx

```diff
@@ -47,34 +63,36 @@ export const onRenderBody = ({ setHeadComponents }) => {
  * A workaround described in https://github.com/gatsbyjs/gatsby/issues/1526 is
  * implemented below
  */
-export const onPreRenderHTML = ({
+export const onPreRenderHTML: GatsbySSR['onPreRenderHTML'] = ({
   getHeadComponents,
   replaceHeadComponents,
 }) => {
```

Comment:
If you type this parameter you will receive the return type from getHeadComponents

Reply:
I think it's working now. Can you re-check?